engine/class_modules/priest/sc_priest_discipline.cpp

```diff
@@ -280,6 +280,12 @@ void priest_t::create_buffs_discipline()
   buffs.shadow_covenant =
       make_buff( this, "shadow_covenant", talents.discipline.shadow_covenant->effectN( 4 ).trigger() )
           ->set_trigger_spell( talents.discipline.shadow_covenant );
+
+  // 280391 has the correct 40% damage increase value, but does not apply it to any spells.
+  // 280398 applies the damage to the correct spells, but does not contain the correct value (12% instead of 40%).
+  // That 12% represents the damage REDUCTION taken from the 40% buff for each attonement that has been applied.
```

Comment:
Maybe note as a `TODO` to add support for Atonements reducing this?

Reply:
Done